src/migrations/1728032163157-dropAllIndexesFKsRelations.ts

```
@@ -0,0 +1,103 @@
+        await queryRunner.query(
+          `DROP INDEX \`${index.INDEX_NAME}\` ON \`${index.TABLE_NAME}\``
+        );
+        console.log(
+          `Dropped index ${index.INDEX_NAME} on table ${index.TABLE_NAME}`
+        );
+      } catch (error) {
+        console.error(
+          `Could not drop index ${index.INDEX_NAME} on table ${index.TABLE_NAME}:`,
+          error
+        );
+        continue;
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)